System/Metrics/Dimensional.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE LambdaCase #-}
```

Comment:
This file needs some top-level Haddocks explaining how to use this module and what it's for.